pkg/errcode/errcode.go

```diff
@@ -160,6 +160,69 @@ const (
 	ErrMeshConfigMarshaling
 )
 
+// Range 5000-5500 reserved for errors related to Envoy
```

Comment:
Do you want to make it more specific, like "related to Envoy xDS server/control plane"?